src/stim/diagram/detector_slice/detector_slice_set.cc

```diff
@@ -615,31 +617,59 @@ void DetectorSliceSet::write_svg_diagram_to(std::ostream &out) const {
 
     for (size_t k = 0; k < num_ticks; k++) {
         for (auto q : used_qubits()) {
-            auto c = coords(min_tick + k, q);
+            auto t = min_tick + k;
+            
+            std::stringstream id_ss;
+            id_ss << "qubitdot";
```

Comment:
Maybe `qubit_indicator`?

Reply:
i was thinking to label it as a visual element rather than abstractly what it does; there's other things that indicate (at) a qubit, but not other thing that is a qubit dot. Similar for qubitline rather than say qubit indicator

Reply:
k